### ISSUE TITLE
[BUG]: can't select the mask to merge or delete in IO

### ISSUE THREAD

**Issue by Devaprasad-reddy** [ankidroid/Anki-Android#15899]
### Checked for duplicates?

- [X] This issue is not a duplicate

### What are the steps to reproduce this bug?

Happened twice randomly couldn't reproduce 

### Expected behaviour

Touching on the mask should select it, so I can modify or merge with another mask

### Actual behaviour

The mask is stuck and not able to click on it or select it with the arrow tool

https://github.com/ankidroid/Anki-Android/assets/36725943/b8d0bb62-d8e1-40f3-a91b-14a07506da9b

### Debug info

```text
AnkiDroid Version = 2.17.6 (568538896be943818ca793dd2635822265712def)

Backend Version = 0.1.34-anki23.12.1 (23.12.1 1a1d4d5419c6b57ef3baf99c9d2d9cf85d36ae0a)

Android Version = 14 (SDK 34)

ProductFlavor = full

Manufacturer = OnePlus

Model = OPD2203

Hardware = mt8798

Webview User Agent = Mozilla/5.0 (Linux; Android 14; OPD2203 Build/UKQ1.230924.001; wv) AppleWebKit/537.36 (KHTML, like Gecko) Version/4.0 Chrome/122.0.6261.66 Safari/537.36

ACRA UUID = 3213579c-ae2d-4097-a4d5-6497675d3034

Crash Reports Enabled = true
```

### Research

- [X] I am reporting a bug specific to AnkiDroid (Android app)
- [X] I have checked the [manual](https://ankidroid.org/docs/manual.html) and the [FAQ](https://github.com/ankidroid/Anki-Android/wiki/FAQ) and could not find a solution to my issue
- [X] (Optional) I have confirmed the issue is not resolved in the latest alpha release ([instructions](https://docs.ankidroid.org/manual.html#betaTesting))

**Comment by krmanik** [comment]
There may be some issues selectable, @Devaprasad-reddy how often this happen? Also does this happen if you switch between sidebar tools.

**Comment by Devaprasad-reddy** [comment]
Happened twice yesterday after the 2.17.6 update.
I often switch tools, this never happened before

**Comment by krmanik** [comment]
The next version of IO have fixes, which will fix the issue.

**Comment by david-allison** [comment]
@krmanik was this fixed in 24.04?

**Comment by krmanik** [comment]
It is fixed.
